Marlin/gcode.h

```diff
@@ -148,6 +169,9 @@ class GCodeParser {
   // Code value pointer was set
   FORCE_INLINE static bool has_value() { return value_ptr != NULL; }
 
+  // Seen and has value
+  FORCE_INLINE static bool seenval(const char c) { return seen(c) && has_value(); }
```

Comment:
Just be careful. The `g26_repeats` code is already badly written, so the fix there is obvious. Generally speaking we should be flagging an error when parameters that require a value are supplied without a value. Ignoring the parameter when it lacks a value might not be the expected behavior in all cases. So… just be selective in its use.